spacemesh/v2alpha1/tx.proto

```diff
@@ -18,22 +40,27 @@ message TransactionV1 {
   string template = 3; // template account address
   uint32 method = 4; // this is actually limited by uint8, but no type for that.
   Nonce nonce = 5;
-  LayerLimits limits = 6;
-  uint64 max_gas = 7; // max gas units consumed by tx
-  uint64 gas_price = 8; // fee per unit gas, in smidge
-  uint64 max_spend = 9;
-  bytes raw = 10;
+  uint64 max_gas = 6; // max gas units consumed by tx
+  uint64 gas_price = 7; // fee per unit gas, in smidge
+  uint64 max_spend = 8;
+  bytes raw = 9;
+  enum TransactionType {
```

Comment:
For the record there are other types, involving the vault and vesting wallet templates, but neither is relevant right now and we can add these later.